stan/math/prim/mat/fun/cov_exp_quad.hpp

```diff
@@ -84,8 +86,8 @@ namespace stan {
                   Eigen::Dynamic, Eigen::Dynamic>
     cov_exp_quad(const std::vector<T_x1>& x1,
                  const std::vector<T_x2>& x2,
-                 T_sigma& sigma,
-                 T_l& l) {
+                 const T_sigma& sigma,
+                 const T_l& l) {
       using std::exp;
```

Comment:
You need it if one of the template parameters can be
instantiated to `double`.  And you should include
the files for stan::math::exp if you haven't.

- Bob


> On Sep 2, 2016, at 8:09 PM, Rob Trangucci <notifications@github.com> wrote:
> 
> In stan/math/prim/mat/fun/cov_exp_quad.hpp:
> 
> > @@ -84,8 +86,8 @@ namespace stan {
> >                    Eigen::Dynamic, Eigen::Dynamic>
> >      cov_exp_quad(const std::vector<T_x1>& x1,
> >                   const std::vector<T_x2>& x2,
> > -                 T_sigma& sigma,
> > -                 T_l& l) {
> > +                 const T_sigma& sigma,
> > +                 const T_l& l) {
> >        using std::exp;
> 
> You're right, I'm actually using stan::math::exp, so I've removed these std::exp calls
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
>